hadoop-yarn-project/hadoop-yarn/hadoop-yarn-client/src/main/java/org/apache/hadoop/yarn/client/api/impl/NMClientImpl.java

```diff
@@ -124,11 +124,11 @@ protected synchronized void cleanupRunningContainers() {
       } catch (YarnException e) {
         LOG.error("Failed to stop Container " +
             startedContainer.getContainerId() +
-            "when stopping NMClientImpl");
+            "when stopping NMClientImpl", e);
```

Comment:
what’s the e user for in log?

Reply:
If there's no "e", the log here will only show the message "Failed to stop Container ### when stopping NMClientImpl". People will only know there's something wrong here, but will not know what's wrong here. (YarnException? IOException?)
If add an "e" here, the exception type and position where this exception happens will also be generated to the log, so that people will know what's wrong here.